v1/message.proto

```diff
@@ -3,6 +3,6 @@ syntax = "proto3";
 package statusproto;
 
 message StatusProtocolMessage {
-  bytes signature = 1;
-  bytes payload = 2;
+  bytes signature = 4001;
```

Comment:
Same here as https://github.com/vacp2p/specs/pull/1 would be good to document this more explicitly

cc @adambabik